0001-typedb/02-dev/01-connections.md

```diff
@@ -0,0 +1,371 @@
+---
+pageTitle: Connections
+keywords: typedb, basics, connect, connection, session, database
+longTailKeywords: basic concepts of typedb, typedb connection, typedb database, typedb session
+summary: Brief description of connection to TypeDB.
+toc: false
+---
+
+# Connections
+
+## Clients
+
+TypeDB server accepts remote connections from a number of [TypeDB Clients](../../02-clients/00-clients.md) via 
+[gRPC](https://en.wikipedia.org/wiki/GRPC) protocol.
+
+Once connected, TypeDB clients can manage [databases](#databases) and [sessions](#sessions).
```

Comment:
> "manage databases [and users], and open sessions to read and write schema and data"?

It can open sessions, or it can close them. Also, it can use sessions to open/close transactions. Session management is actually quite inclusive here.